src/pages/task/TestsTableCore.tsx

```diff
@@ -132,7 +122,7 @@ const columns: Array<ColumnProps<TaskTestsData>> = [
     }
   }
 ];
-const rowKey = ({ id }: { id: string }): string => id;
+export const rowKey = ({ id }: { id: string }): string => id;
```

Comment:
should this be exported from a util file since it's being used by another page?

Reply:
yeah could do that. was also thinking it might be safer to just re-declare it in the Tasks Table component. that way tables are not susceptible to any changes in other components.